rules/windows/process_creation/win_malware_notpetya.yml

```diff
@@ -24,12 +24,12 @@ logsource:
     product: windows
 detection:
     pipe_com:
-        CommandLine: '*\AppData\Local\Temp\\* \\.\pipe\\*'
+        CommandLine|contains: '\AppData\Local\Temp\\* \\.\pipe\\'
```

Comment:
need to reconstruct it in a way to get rid of asterisks as we are using value modifier here

Reply:
Fixed